python-black.sublime-settings

```diff
@@ -1,10 +1,23 @@
 {
-    // Whether to automatically format the entire document when saving.
-    // There are three modes:
-        // - true (deprecated)
-        // - false (deprecated)
-        // - "on"
-        // - "off"
-        // - "smart": Automatic formatting is only enabled if there is a `black` section in the project's `pyproject.toml`
-    "format_on_save": "on"
+   // Whether to automatically format the entire document when saving.
+   // There are three modes:
+   //    - true (deprecated)
+   //    - false (deprecated)
+   //    - "on"
+   //    - "off"
+   //    - "smart": Automatic formatting is only enabled if there is a `black` section in the project's `pyproject.toml`
+   "format_on_save": "on",
+   // Black [OPTIONS]
+   "options": {
+      "target_version": [],
+      "line_length": 88,
+      "string_normalization": true,
+      "is_pyi": false,
+      "is_ipynb": false,
+      "skip_source_first_line": false,
+      "magic_trailing_comma": true,
+      "experimental_string_processing": false,
+      "python_cell_magics": [],
+      "preview": false
```

Comment:
These keys are not supported for this project:

- is_ipynb
- preview

They should be removed in optional.

Reply:
These two options have been removed.
Besides, the `python_cell_magics` is related to `is_ipynb` for jupyter notebook, and `experimental_string_processing has been included to 'preview' and will be deprecated, so they're also removed.